pkg/metronome/client.go

```diff
@@ -421,11 +440,14 @@ func (c *Client) Queued(jobID string) ([]Queue, error) {
 			}
 		}
 		return nil, fmt.Errorf("job '%s' does not exist", jobID)
+	case 404:
+		return nil, fmt.Errorf("job %s does not exist", jobID)
```

Comment:
Line 442: `fmt.Errorf("job '%s' does not exist", jobID)`
Here: `fmt.Errorf("job %s does not exist", jobID)`
Can we make those two messages consistent?

Reply:
Ok I'll go over all of them to make them consistent.